coldfront/core/allocation/management/commands/create_allocation_periods.py

```diff
@@ -163,6 +163,11 @@ def get_allocation_periods():
         if flag_enabled('LRC_ONLY'):
             periods.extend(
                 [
+                    {
```

Comment:
This addition doesn't seem relevant/necessary.

Reply:
Okay, I do have this on my LRC box and on the staging instance. I added a zero before the month to be consistent with everything else. Thanks!